dicomsort.py

```diff
@@ -27,8 +27,9 @@
 import zipfile
 
 # special public packages
-import dicom
-from dicom.filereader import InvalidDicomError
+import pydicom
+
+from pydicom.filereader import InvalidDicomError
```

Comment:
How about something like:

try:
  import dicom
  pydicom = dicom
except ImportError:
  import pydicom

Just in case someone is using it with older versions

Reply:
Yes, that's similar to what I tried, but somehow it was still erroring in `dicom.filereader` import. I will add that inside the try/catch.